explainaboard/metrics/accuracy.py

```diff
@@ -39,6 +39,19 @@ class CorrectCount(Accuracy):
     Calculate the absolute value of correct number
     """
 
+    def calc_stats_from_data(
+        self, true_data: list, pred_data: list, config: Optional[MetricConfig] = None
+    ) -> MetricStats:
+
+        return MetricStats(
+            np.array(
+                [
+                    (1.0 if y == x or y in str(x).split("/") else 0.0)
```

Comment:
Instead of doing this splitting it might be better if the original dataset stored the multiple correct answers as a list?